docs/docs/user-manuals/how-to-use-frontend.md

```diff
@@ -0,0 +1,237 @@
+---
+sidebar_position: 1
+---
+
+import CustomImage from "@site/src/components/CustomImage";
+
+# How to Use the Frontend
+
+## Version to which this Document Refers
+
+To be precise, [this is the latest commit](https://github.com/MiSArch/frontend/commit/c7b74ec563058a66df03108ba27249450a00ce7e) of the Frontend version that was used when creating this document.
+
+## Purpose of this Document
+
+This document is mainly a step-by-step guide detailing how to **use** the Frontend. If you have just deployed the system and do not know where to start, this guide is for you. It tells you step by step how to manage the online store and how to use it.
+Screenshots are provided in the final section of this document. Why? Because we recommend using this document alongside the actual frontend, side by side, when following the steps. We believe that inserting screenshots within the text would be distracting and unnecessarily bloat the document.
+
+## Disclaimer
+
+The Frontend is still in development and therefore this document refers to exactly one explicit version of the Frontend. It may happen that the document becomes outdated because the Frontend has advanced in the meantime. Ideally, with each new feature or bug fix, this document is updated accordingly.
+
+## Where to Find the Frontend
+
+If we deploy the MiSArch system — the whole infrastructure — on our local machine we will find the Frontend at the following URL: http://localhost:4000.
+
+## Step-by-step User Guide
+
+This section primarily describes step-by-step how to operate the Frontend so that each feature can be tried out. This is because some features are only available if you have exercised another feature in advance. For example, you have to have created a product before you can order it in the online store. Let's start with user management. This is where Keycloak comes into play. Once we know how to register a user and assign them permissions to manage the online store, we move on to managing the online store — how do we add products? How do we restock the products? How do we create tax rates? In the third subsection, we then use the frontend from the customer's perspective.
```

Comment:
```suggestion
This section primarily describes step-by-step how to operate the Frontend so that each feature can be tried out. This is because some features are only available if you have executed something else previously.
For example, you must have created a product before you can order it in the online store.
Let's start with user management.
This is where Keycloak comes into play.
Once we know how to register a user and assign them permission to manage the online store, we move on to managing the online store — how do we add products? How do we restock the products? How do we create tax rates? In the third subsection, we then use the frontend from the customer's perspective.
```